### PR TITLE
Add search e2e test with screenshot

### PR DESCRIPTION
**Pull request by florkbr**
Resolves https://issues.redhat.com/browse/RHCLOUD-26170

Example generated screenshot:
Screenshot 2023-08-24 at 6 36 52 PM

### CODE REVIEW

**Comment by florkbr** [comment]
@karelhala it looks like we run the component cypress E2E tests (`insights-chrome/cypress/component`) in both the github PR checks and the travis pipeline. Is that necessary? May be able to save some time in our PR checks.

**Comment by florkbr** [comment]
Getting gateway timeouts on the github PR connector in the devshift.net pipeline 😕

**Comment by Hyperkid123** [comment]
> @karelhala it looks like we run the component cypress E2E tests (`insights-chrome/cypress/component`) in both the github PR checks and the travis pipeline. Is that necessary? May be able to save some time in our PR checks.

The idea is to transition all to the Jenkins from the gh-actions and travis. The Jenkins work is currently WIP and the results are not 100% accurate. That is why we have duplicates all over the place. It should be resolved shortly.

**Review by karelhala — APPROVED** [review]

File: cypress/e2e/release-gate/search.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      {
+        method: 'GET',
+        url: '**/hydra/rest/search/**',
+      },
+      searchResponse
+    ).as('search');
+    cy.get('.chr-c-search__input').click().type('openshift');
+    cy.wait('@search').its('response.statusCode').should('equal', 200);
+    cy.get('@search.all').should('have.length', 1);
+    cy.screenshot();
```
Comment (Hyperkid123) [inline cypress/e2e/release-gate/search.cy.tsx:63]: There is an image snapshot command that does compare screenshots (similar to Jest snapshots). Is that what you wanted to achieve? The `cy.screenshot` is ignored.

**Pull request merged**